.nvmrc

```diff
@@ -0,0 +1 @@
+10
```

Comment:
Please add a new line at the end of the file.

Reply:
@data-get updated. Marking as resolved. Thanks for the feedback!